dockerfile-generator.rb

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env ruby
```

Comment:
I guess I don't understand?

can you describe the struggles you were having with the regular dockerfile?